src/django_elasticsearch_dsl_drf/filter_backends/search/query_backends/nested.py

```diff
@@ -75,15 +75,21 @@ def construct_search(cls, request, view, search_backend):
                     for _field in options.get('fields', []):
                         # In case if we deal with structure 2
                         if isinstance(_field, dict):
-                            # TODO: take options (ex: boost) into consideration
+                            # take options (ex: boost) into consideration
+                            field_options = {key: value for key, value in _field.items() if key != 'name'}
                             field = "{}.{}".format(path, _field['name'])
+                            field_kwargs = {
+                                field: {
```

Comment:
Look good, except that I don't want to drop Python 2.7 in a minor release just yet. Could you replase this with Python 2.7 compatible structure? Like, ``field_options.update({"query": search_term})`` and then ``field: field_options``.

Reply:
@bashu done